test_runner/src/main/kotlin/ftl/reports/xml/JUnitXml.kt

```diff
@@ -6,20 +6,22 @@ import com.fasterxml.jackson.module.kotlin.KotlinModule
 import com.fasterxml.jackson.databind.DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES
 import ftl.reports.xml.model.JUnitTestResult
 import ftl.reports.xml.model.JUnitTestSuite
+import ftl.reports.xml.preprocesor.fixHtmlCodes
 import java.io.File
 import java.nio.file.Files
 import java.nio.file.Path
 
 private val xmlModule = JacksonXmlModule().apply { setDefaultUseWrapper(false) }
+
 private val xmlMapper = XmlMapper(xmlModule)
     .registerModules(KotlinModule())
     .configure(FAIL_ON_UNKNOWN_PROPERTIES, false)
 
 internal val xmlPrettyWriter = xmlMapper.writerWithDefaultPrettyPrinter()
 
-private fun xmlBytes(path: Path): ByteArray {
+private fun xmlText(path: Path): String {
     if (!path.toFile().exists()) throw RuntimeException("$path doesn't exist!")
-    return Files.readAllBytes(path)
+    return String(Files.readAllBytes(path))
```

Comment:
I'd like to merge https://github.com/Flank/flank/pull/767 before merging this pull request.

Reply:
> I'd like to merge #767 before merging this pull request.

Done